src/Service.Host/client/src/components/reports/ChangeStatusReportOptions.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
+import { InputField, Page, Title } from '@linn-it/linn-form-components-library';
+import history from '../../history';
+import config from '../../config';
+
+function ChangeStatusReportOptions() {
+    const [options, setOptions] = useState({
```

Comment:
seems kind of pointless to have an options object if there is only one option.
const [months, setMonths] = useState(3) 
would be better